playwright/data/addresses.ts

```diff
@@ -0,0 +1,59 @@
+import faker from "faker";
```

Comment:
please stop using old faker in the PW code, it is not maintained anymore, consider even migrating in your PW tests (as this is the new codebase) to https://fakerjs.dev/ (the new branch of it)

Reply:
Will make the refactor implementing new faker in a separate PR